config/crd/kustomization.yaml

```diff
@@ -2,4 +2,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://github.com/kong/kubernetes-configuration/config/crd/ingress-controller?ref=v1.1.0 # Version is auto-updated by the generating script.
+  - https://github.com/kong/kubernetes-configuration/config/crd/ingress-controller?ref=v1.1.1-0.20250224133355-0c2aa5d85cad # Version is auto-updated by the generating script.
```

Comment:
With small tweaks to the code generating script (to make it work with SHAs) there's no need for tags on the dependency.

ref: https://github.com/Kong/gateway-operator/pull/1224

Reply:
cc: @czeslavo

Reply:
Done in 6501ec93eca34e3e1ac5bb675cf7dda35812f98a